diff -Naurp tvheadend-77fc1c0/src/webui/webui.c tvheadend-77fc1c0-patch/src/webui/webui.c
--- tvheadend-77fc1c0/src/webui/webui.c	2018-10-30 11:09:17.000000000 +0200
+++ tvheadend-77fc1c0-patch/src/webui/webui.c	2018-10-30 11:14:37.000000000 +0200
@@ -519,7 +519,7 @@ http_m3u_playlist_add(htsbuf_queue_t *hq
   htsbuf_append_str(hq, "#EXTINF:-1");
   if (logo) {
     if (strncmp(logo, "imagecache/", 11) == 0) {
-      htsbuf_qprintf(hq, " logo=\"%s/%s", hostpath, logo);
+      htsbuf_qprintf(hq, " tvg-logo=\"%s/%s", hostpath, logo);
       switch (urlauth) {
       case URLAUTH_NONE:
         break;
@@ -533,7 +533,7 @@ http_m3u_playlist_add(htsbuf_queue_t *hq
       }
       htsbuf_append_str(hq, "\"");
     } else {
-      htsbuf_qprintf(hq, " logo=\"%s\"", logo);
+      htsbuf_qprintf(hq, " tvg-logo=\"%s\"", logo);
     }
   }
   if (epgid)
